### ISSUE TITLE
Deleted posts in search results

### ISSUE THREAD

**Issue by davad** [KyleAMathews/Journal#8]
Deleted results are showing up in test results.

I have some test posts I made when I first setup Journal. I searched "test" and had them show up.

**Comment by KyleAMathews** [comment]
Yeah, this is a known issue. The fix would be pretty easy. Just add a function to delete a post and call that in the DELETE post REST call.
